File: app/src/main/java/com/mikepenz/materialdrawer/app/drawerItems/CustomBaseViewHolder.java
package com.mikepenz.materialdrawer.app.drawerItems;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.mikepenz.materialdrawer.app.R;

class CustomBaseViewHolder extends RecyclerView.ViewHolder {
    protected View view;
    protected ImageView icon;
    protected TextView name;
    protected TextView description;

    public CustomBaseViewHolder(View view) {
        super(view);

        this.view = view;
        this.icon = (ImageView) view.findViewById(R.id.material_drawer_icon);
        this.name = (TextView) view.findViewById(R.id.material_drawer_name);
        this.description = (TextView) view.findViewById(R.id.material_drawer_description);
    }
}

File: app/src/main/java/com/mikepenz/materialdrawer/app/drawerItems/CustomBaseViewHolder.kt
package com.mikepenz.materialdrawer.app.drawerItems

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.mikepenz.materialdrawer.app.R

open class CustomBaseViewHolder(var view: View) : RecyclerView.ViewHolder(view) {
    var icon: ImageView = view.findViewById<ImageView>(R.id.material_drawer_icon)
    var name: TextView = view.findViewById<TextView>(R.id.material_drawer_name)
    var description: TextView = view.findViewById<TextView>(R.id.material_drawer_description)
}